src/extension.ts

```diff
@@ -174,15 +205,28 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
         javaPath,
         classpath,
         serverProperties,
-        javaOptions
+        javaOptions,
+        customRepositories
       );
     },
-    () => {
+    (reason: any) => {
       const msg = (() => {
         const proxy =
           `See https://scalameta.org/metals/docs/editors/vscode.html#http-proxy for instructions ` +
           `if you are using an HTTP proxy.`;
-        if (process.env.FLATPAK_SANDBOX_DIR) {
+        if (reason.code == 1) {
```

Comment:
```suggestion
        if (reason.code === 1) {
```

Minor, but I always use `===` unless there's a good reason not to.